docs/_docs/user-docs/recipe-headers.md

```diff
@@ -251,6 +251,80 @@ The above would remove everything except for the pixel data, and a few fields
 that are relevant to its dimensions. It would add a field to indicate the
 patient's identity was removed.
 
+The table below shows the full details of how multiple actions defined on the same
```

Comment:
@vsoch - As an additional check, I ran the action interaction tests against 0.2.28.  While I expected the tests that involved KEEP and BLANK to fail, there were a few others which had altered behavior.  The image below is the 0.3.1 action interaction grid added to the documentation.

- The cells in green indicate those tests which functioned the same in 0.2.28 vs 0.3.1.
- The cells in yellow indicate the tests which run successfully in 0.3.1, but **expectedly** fail in 0.2.28 due to KEEP/BLANK issues.
- The cells in red are those that run successfully in 0.3.1, but fail in 0.2.28.

![image](https://user-images.githubusercontent.com/32198293/204913116-5ecd540e-94d1-4af7-a24a-ebea2994953e.png)

Given the "precedence" defined, I think that the cells in red are actually okay too, since per the precedence of the actions, JITTER and REPLACE are both > REMOVE.   While this interaction has changed since 0.2.28, I believe this is a correction in behavior. 

I'd just like to make sure that I'm thinking correctly on all of this and we don't have a bug that needs to be addressed.

Reply:
This is so pretty! So the table is saying when you do a "thing in row index" then a "thing in column index" there is some underlying test that is successful / failed / no change. So for the red ones that means:

- Jitter then remove has changed functionality
- Replace then remove has changed functionality
- Remove then jitter and remove then replace is changed.

The last row makes sense - if Jitter and replace are preferred over remove, then we shouldn't see them removed. And then the opposite pattern for the other two. So yes I think this makes sense.